demo-transactions/src/main/kotlin/io/holixon/cqrshexagonaldemo/demoparent/transactions/command/application/port/outbound/account/AccountService.kt

```diff
@@ -8,4 +8,5 @@ interface AccountService {
     fun findAccount(iban: Iban): Account
     fun createAccount(account: Account): Account
     fun deposit(account: Account, amount: Amount): Account
+    fun withdraw(account: Account, amount: Amount): Account
```

Comment:
We are mixing up things again here. This is a repo, it should not do modifications of the account object, just persist or load.
Maybe the interface name isn't speaking enough - maybe we should rename it to `AccountRepo` or `AcccountPersistenceService` or alike.

`deposit` and `withdraw` are to be moved into either a domain service or the `Account` domain model itself.